test/lib/minion_test.exs

```diff
@@ -39,5 +39,60 @@ defmodule AssembleTheMinions.MinionTest do
     assert kevin_count == 2
   end
 
+  test "Can add to the count" do
+    AssembleTheMinions.Minion.start_link(:jorge)
+    AssembleTheMinions.Minion.add(:jorge, 2)
+    assert 2 = AssembleTheMinions.Minion.current_count(:jorge)
```

Comment:
Generally `==` for assertions.  This works because pattern matching 2 against 2 works, but for other things it will break